src/experiment_prototype/experiment_slice.py

```diff
@@ -749,85 +749,83 @@ def check_pulse_phase_offset(cls, ppo, values):
                     )
         return ppo
 
-    @validator("txctrfreq", always=True)
-    def check_txctrfreq(cls, txctrfreq, values):
-        if not txctrfreq:
-            txctrfreq = 12000.0  # Default value when not set
-
-        # Note - txctrfreq set here and modify the actual center frequency to a
-        # multiple of the clock divider that is possible by the USRP - this default value set
-        # here is not exact (center freq is never exactly 12 MHz).
-
-        # convert from kHz to Hz to get correct clock divider. Return the result back in kHz.
-        clock_multiples = options.usrp_master_clock_rate / 2**32
-        clock_divider = math.ceil(txctrfreq * 1e3 / clock_multiples)
-        txctrfreq = (clock_divider * clock_multiples) / 1e3
+    @validator("txctrfreq", always=True, pre=True)
+    def check_txctrfreq(cls, txctrfreq):
+        if isinstance(txctrfreq, (float, int)):
+            # Note - txctrfreq set here and modify the actual center frequency to a
+            # multiple of the clock divider that is possible by the USRP - this default value set
```

Comment:
Is this true? It doesn't look like its set here any longer, just modified slightly

Reply:
Good catch. Completely forgot to remove that comment.